### PR TITLE
docs: reorder & regroup permutation groups chapter

### PR DESCRIPTION
**Pull request by fingolfin**

### CODE REVIEW

File: docs/src/Groups/permgroup.md [hunk 1]
```diff
@@ -30,12 +31,38 @@ projective_unitary_group
 projective_special_unitary_group
 ```
 
+## Operations for permutation groups
+
+All operations, properties and attributes for general groups
+describe in the previous sections are supported for permutation
```
Comment (ThomasBreuer) [inline docs/src/Groups/permgroup.md]: described

File: docs/src/Groups/permgroup.md [hunk 2]
```diff
@@ -30,12 +31,38 @@ projective_unitary_group
...
+groups. In addition there are some specific to permutation group.
```
Comment (ThomasBreuer) [inline docs/src/Groups/permgroup.md]: to permutation groups

**Review by ThomasBreuer — APPROVED** [review]
Fine with me.
Perhaps fix the two typos.

What is the aim of separate `@docs` groups without introductory text for each of them?

File: docs/src/Groups/permgroup.md (hunk 1 above)
Comment (lgoettgens) [reply docs/src/Groups/permgroup.md]:
```suggestion
described in the previous sections are supported for permutation
```

File: docs/src/Groups/permgroup.md (hunk 2 above)
Comment (lgoettgens) [reply docs/src/Groups/permgroup.md]:
```suggestion
groups. In addition there are some specific to permutation groups.
```

File: docs/src/Groups/permgroup.md [hunk 3]
```diff
@@ -30,12 +31,38 @@ projective_unitary_group
...
+
 In OSCAR, every permutation group has a degree `n`, that corresponds to the size of the set on which `G` acts.
 
 ```@docs
 degree(x::PermGroup)
 ```
 
+```@docs
```
Comment (fingolfin) [inline docs/src/Groups/permgroup.md:48]: The reason there are three `@docs` blocks here is mundane... I copied them here from elsewhere as the changes in this PR evolved. I also thought about merging them, but then the separation seemed to make some sense. Perhaps some text could be added between them to make this clear, e.g.:
````suggestion

The following functions deal with the natural action of a given permutation group $G$.
```@docs
````

File: docs/src/Groups/permgroup.md [hunk 4]
```diff
@@ -30,12 +31,38 @@ projective_unitary_group
...
+is_regular(G::PermGroup, L::AbstractVector{Int} = 1:degree(G))
+is_semiregular(G::PermGroup, L::AbstractVector{Int} = 1:degree(G))
+rank_action(G::PermGroup, L::AbstractVector{Int} = 1:degree(G))
+blocks(G::PermGroup, L::AbstractVector{Int} = moved_points(G))
+maximal_blocks(G::PermGroup, L::AbstractVector{Int} = moved_points(G))
+minimal_block_reps(G::PermGroup, L::AbstractVector{Int} = moved_points(G))
+all_blocks(G::PermGroup)
+```
+
+```@docs
```
Comment (fingolfin) [inline docs/src/Groups/permgroup.md:63]:
````suggestion
The following functions allow efficiently "recognizing" certain permutation groups as alternating or symmetric groups.

```@docs
````

File: docs/src/Groups/permgroup.md (hunk 3 above)
Comment (fingolfin) [reply docs/src/Groups/permgroup.md:48]: To be clear, I am neutral as to whether we add these (or other) filler texts; or fuse the `@docs` blocks; or leave them as they are in this PR. If anyone has a preference either way, please say so.

Comment (ThomasBreuer) [reply docs/src/Groups/permgroup.md:48]: I like the new filler texts.

**Pull request merged**